src/components/GoalSuggest.tsx

```diff
@@ -117,9 +118,9 @@ export const GoalSuggest = forwardRef<HTMLDivElement, GoalSuggestProps>(
                 value={value}
                 onChange={onSelectItem}
                 items={items}
-                visible={visible}
+                visible={visible && showSuggest}
                 renderInput={renderInput}
-                maxWidth={400}
+                maxWidth={600}
```

Comment:
maybe we should make this relative of some sort?

Reply:
no, 'cause this property of Popup lib component, which calculating based on content width and value of prop